src/styles/App.css

```diff
@@ -0,0 +1,357 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.App {
+  background-color: #f5f6fa;
+  padding: 3.75rem;
+}
+
+.header {
+  height: 5.938rem;
+}
+
+.nav {
+  background-color: white;
+  height: 100%;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 6.25rem;
+}
+
+.navbar {
+  display: flex;
+  list-style: none;
+  gap: 2.5rem;
+  align-items: center;
+}
+
+.logo {
+  text-decoration: none;
+  font-size: 1.875rem;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  color: #0290ff;
+}
+
+.nav-link {
+  text-decoration: none;
+  font-family: Montserrat, sans-serif;
+  font-size: 0.813rem;
+  font-weight: normal;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: 1.9px;
+  color: #121212;
+  opacity: 0.5;
+}
+
+.active-link {
+  text-decoration: none;
+  font-family: Montserrat, sans-serif;
+  font-size: 0.813rem;
+  font-weight: normal;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: 1.9px;
+  color: #121212;
+}
```

Comment:
as seen in many cases in your CSS file, you use the same rule for multiple elements which is not a CSS best practice. please create a rule for multiple elements using class attributes 

eg 
```
.font {
text-decoration: none;
  font-weight: normal;
  font-stretch: normal;
  font-style: normal;
  line height: normal;
} 

.font-family {
font-family: Montserrat, sans-serif;
}
```

N.B
## Use a CSS Table of Contents
#### It is recommended to use a CSS table of contents when starting your style sheets. This will help you track down your code easily as you move along. Keep things organized and you will write styles more efficiently.

Consider this example below:

```
/* Project Meta */

/* reset  */

/* general */

/* typography */

/* grid */

/* header */

/* footer */

/* Page template A */

/* Page template B */

/* Media Queries */
```

Reply:
Thanks for your review @alexander16108, I will gladly fix it 👍